### ISSUE TITLE
%pdb followed by error crashes IPython

### ISSUE THREAD

**Issue by cottrell** [ipython/ipython#12040]
Basically you run the %pdb magic and then type something like "asdf" and it crashes.

```
site-packages/IPython/terminal/debugger.py", line 69, in pt_init
    options['inputhook'] = self.inputhook
```

The problem occurs with 7.10.1 but not with 7.10.0. I'm on ubuntu python 3.7.3

**Comment by cottrell** [comment]
Seems like this commit 

f52ca01cad84187a87ee0159778886b3764dbb18

**Comment by cottrell** [comment]
This is fixed in master by bcc84fb01a46f660c08f004842cbd7be3b6c8441
